modules/private/repository.go

```diff
@@ -44,7 +44,7 @@ func GetRepository(repoID int64) (*models.Repository, bool, error) {
 
 // ActivePullRequest returns an active pull request if it exists
 func ActivePullRequest(baseRepoID int64, headRepoID int64, baseBranch, headBranch string) (*models.PullRequest, error) {
-	reqURL := setting.LocalURL + fmt.Sprintf("api/internal/active-pull-request?baseRepoID=%d&headRepoID=%d&baseBranch=%s&headBranch=%s", baseRepoID, headRepoID, url.QueryEscape(baseBranch), url.QueryEscape(headBranch))
+	reqURL := setting.LocalURL + fmt.Sprintf("api/internal/active-pull-request?baseRepoID=%d&headRepoID=%d&baseBranch=%s&headBranch=%s", baseRepoID, headRepoID, util.PathEscapeSegments(baseBranch), util.PathEscapeSegments(headBranch))
```

Comment:
These ones are actually in a query so `url.QueryEscape` is the correct thing.

Reply:
done